### PR TITLE
Add enable-nota attribute for <pl-multiple-choice> element

### PR DESCRIPTION
**Pull request by bojinyao**
In response to: https://github.com/PrairieLearn/PrairieLearn/issues/2600

`enable-nota` will be an additional attribute to `<pl-multiple-choice>` element of type boolean, default to false

When set to true:
- An option `None of the above` will always appear at the *bottom* of all sampled choices
    - This option will be correct with probability `1/(number_correct + 1)`
    - If no correct answer is provided, the option will be correct with probability 1
- The total number of choices is still upper-bounded by `number-answers` attribute.
- `fixed-order` does not affect its placement
- This has no meaningful effect on `inline`, `weight`, or `answers-name` attributes

### CODE REVIEW

File: elements/pl-multiple-choice/pl-multiple-choice.py [hunk 1]
```diff
@@ -11,9 +11,10 @@
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     required_attribs = ['answers-name']
-    optional_attribs = ['weight', 'number-answers', 'fixed-order', 'inline']
+    optional_attribs = ['weight', 'number-answers', 'fixed-order', 'inline', 'enable-nota']
```
Comment (nicknytko) [inline elements/pl-multiple-choice/pl-multiple-choice.py:14]: Can we name the attribute `none-of-the-above` to be more consistent with the others?

File: elements/pl-multiple-choice/pl-multiple-choice.py [hunk 2]
```diff
@@ -11,9 +11,10 @@
...
     pl.check_attribs(element, required_attribs, optional_attribs)
     name = pl.get_string_attrib(element, 'answers-name')
+    enable_nota = pl.get_boolean_attrib(element, 'enable-nota', False)
```
Comment (nicknytko) [inline elements/pl-multiple-choice/pl-multiple-choice.py:17]: Lets add a `NONE_OF_THE_ABOVE_DEFAULT = False` constant to the top of the file and reference it here

File: elements/pl-multiple-choice/pl-multiple-choice.py (hunk 1 above)
Comment (bojinyao) [reply elements/pl-multiple-choice/pl-multiple-choice.py]: updated 👍

File: elements/pl-multiple-choice/pl-multiple-choice.py (hunk 2 above)
Comment (bojinyao) [reply elements/pl-multiple-choice/pl-multiple-choice.py]: updated 👍

**Comment by nicknytko** [comment]
Subsumed by #2623

**Pull request closed without merging**